site/docs/sql/intro.md

```diff
@@ -1,351 +1,399 @@
+  (1, 'ABCD', 5.5, 10),
+  (2, 'EFGH', 14, 20);
+OK
+sql> SELECT * FROM tradeMap;
++---------------+--------------------+----------+--------------------+
+|             id|ticker              |     price|              amount|
++---------------+--------------------+----------+--------------------+
+|              2|EFGH                |14.000000…|                  20|
+|              1|ABCD                |5.5000000…|                  10|
++---------------+--------------------+----------+--------------------+
+2 row(s) selected
+sql>
 ```
 
-It will put the fields from the Kafka topic into the IMap, replicating
-the `ticker` twice: once into the value and once into the key. Since we
-use the `ticker` as the key, every new trade for the same ticker will
-overwrite the previous trade for that ticker, hence the name
-`latest_trades`.
-
-### Creating a Long-Running Job
-
-However, you cannot directly execute the above command because it would
-never complete. Remember, it's reading from a Kafka topic, which is a
-stream of messages. Since it doesn't return any rows to the client, you
-must create a job for it:
+To cancel the job, use:
 
 ```sql
-CREATE JOB trades_ingestion AS
-SINK INTO latest_trades(__key, ticker, price, amount)
-SELECT ticker, ticker, price, amount
-FROM trades
+DROP JOB ingest_trades
 ```
 
-The part after the `AS` keyword is the same as the previous statement.
-The command will create a job named `trades_ingestion`. Now, even if the
-client disconnects, the cluster will continue running the job.
+## Use Jet SQL Embedded Inside Your App
 
-### Checking the Target IMap Contents
+If you use Jet as an embedded library, besides the `hazelcast-jet`
+dependency add also the `hazelcast-jet-sql` dep:
 
-To add some messages to the Kafka topic you can use this command:
+<!--DOCUSAURUS_CODE_TABS-->
 
-```sql
-INSERT INTO trades
-VALUES ('ABCD', 5.5, 10), ('EFGH', 14, 20) /* ,  ... */
-```
+<!--Gradle-->
 
-While this works, we intend to use this way only for prototyping or
-ad-hoc work. Jet is optimized for larger or long-running queries. Under
-the hood, it will spin up a distributed job for each INSERT statement.
-If you benchmarked the number of records per second, the performance
-would be embarrassing.
+```groovy
+compile 'com.hazelcast.jet:hazelcast-jet-sql:{jet-version}'
+```
 
-To check out the contents of the `latest_trades` IMap, use:
+<!--Maven-->
 
-```sql
-SELECT * FROM latest_trades
+```xml
+<dependency>
+    <groupId>com.hazelcast.jet</groupId>
+    <artifactId>hazelcast-jet-sql</artifactId>
+    <version>{jet-version}</version>
+</dependency>
 ```
 
-### Cancelling the Job
+<!--END_DOCUSAURUS_CODE_TABS-->
 
-To cancel the job, use:
+Submit a query:
 
-```sql
-DROP JOB trades_ingestion
+```java
+JetInstance jet = Jet.newJetInstance();
+String query = "select * from TABLE(generate_series(1,5))";
+try (SqlResult result = jet.getSql().execute(query)) {
+    for (SqlRow row : result) {
+        System.out.println(row);
+    }
+}
```

Comment:
It will be nice to add what this program will print. I prefer to look at the output rather than copy-paste the program myself and try it.